packages/terra-form-select/src/Select.scss

```diff
@@ -0,0 +1,149 @@
+:local {
+  .select {
+    background-color: #fff;
+    background-image: var(--terra-select-focus-animation-background, none);
+    background-position: 0 100%;
+    background-repeat: no-repeat;
+    background-size: var(--terra-select-focus-background-start, 0 0);
+    border-color: var(--terra-select-input-border-color, #dedfe0);
+    border-radius: 3px;
+    border-style: solid;
+    border-width: 1px;
+    cursor: pointer;
+    font-size: var(--terra-select-font-size, 1em);
+    line-height: var(--terra-select-line-height, 1.07);
+    padding: var(--terra-select-input-padding, 0.571em 0.5em);
```

Comment:
minor default theme update:
var(--terra-select-input-padding, 0.393rem 0.429rem);

Reply:
new correction - adjusted line-height so now padding had to adjust slightly:
`padding: var(--terra-select-input-padding, 0.3572rem 0.429rem);`